components/function-controller/config/crd/bases/serverless.kyma-project.io_functions.yaml

```diff
@@ -18,6 +16,11 @@ spec:
     description: Runtime is the programming language used for a function e.g. nodejs8
     name: Runtime
     type: string
+  - JSONPath: .spec.visibility
+    description: Visibility defines function's visibility through cluster, available
```

Comment:
ATM visibility may be set only to `cluster-local`. It would be good to have it in description.

> Visibility defines function's visibility through cluster (current version supports only in-cluster visibility)

Reply:
I removed it from CRDs :)